dojo/unittests/tools/test_qualys_parser.py

```diff
@@ -31,3 +31,17 @@ def test_parse_file_with_multiple_vuln_has_multiple_findings(self):
         self.assertEqual(
             finding.unsaved_endpoints[0].host, "demo13.s02.sjc01.qualys.com"
         )
+
+        for finding in findings:
+            if finding.title == "QID-370876 | AMD Processors Multiple Security Vulnerabilities (RYZENFALL/MASTERKEY/CHIMERA-FW/FALLOUT)":
+                break
+        self.assertEqual(
+            finding.unsaved_endpoints[0].host, "demo14.s02.sjc01.qualys.com"
+        )
+        self.assertEqual(
+            # CVSS_FINAL is defined without a cvssv3 vector
+            finding.cvssv3, None
+        )
+        self.assertEqual(
+            finding.severity, "High"
+        )
```

Comment:
Could you add a check that use the new way to gather the vector?

Reply:
good point, found a bug with the new test